src/connectors/metamask.ts

```diff
@@ -28,15 +28,16 @@ export default class MetaMaskConnector extends CKBConnector {
 
   constructor() {
     super();
-    const { publicClient, webSocketPublicClient } = configureChains(
-      [mainnet],
-      [publicProvider()],
-    );
-    this.config = createConfig({
-      autoConnect: true,
-      publicClient,
-      webSocketPublicClient,
-    });
+    // const { publicClient, webSocketPublicClient } = configureChains(
+    //   [mainnet],
+    //   [publicProvider()],
+    // );
+    // this.config = createConfig({
+    //   autoConnect: true,
+    //   publicClient,
+    //   webSocketPublicClient,
+    // });
```

Comment:
I don’t quite understand why this is done. 
If you want to disable MetaMask, just remove `MetaMaskConnector` in `connectors: [new MetaMaskConnector, new JoyIdConnector()]`

Reply:
remove this due to the hotel network not stable, will fix